app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -48,6 +48,8 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
     private val TAG_ABOUT_FRAGMENT = "AboutUsFragment"
     private val TAG_HELP_FRAGMENT = "HelpFragment"
     private val TAG_PRIVACY_FRAGMENT = "PrivacyFragment"
+    private val TAG_CONNECTED_DEVICE_FRAGMENT = "ConnectedDeviceFragment"
+    private val TAG_DEVICE_CONNECT_FRAGMENT = "DeviceConnectFragment"
```

Comment:
When you duplicate the variable, the entire point of making it a constant is lost